mix.exs

```diff
@@ -1,16 +1,21 @@
 defmodule Tabula.Mixfile do
   use Mix.Project
 
+  @version "2.1.0"
+
   def project do
     [app: :tabula,
-      version: "2.1.0",
+      version: @version,
       elixir: "~> 1.3",
       build_embedded: Mix.env == :prod,
       start_permanent: Mix.env == :prod,
       description: "Pretty printer for maps/structs collections",
       package: package(),
       deps: deps(),
-      docs: [extras: ["README.md"]]]
+      docs: [source_ref: "v#{@version}",
+             source_url: "https://github.com/aerosol/Tabula",
```

Comment:
this way we'll be able to link from docs to source code. However, we need v2.1.0 tag published :)